yarn-project/ethereum/src/contracts/rollup.ts

```diff
@@ -179,6 +179,11 @@ export class RollupContract {
     return this.rollup.read.getDepositAmount();
   }
 
+  @memoize
+  getExitDelay() {
```

Comment:
memoize means this wont update if we do call the update function on the smart contract